src/app/app-routing.module.ts

```diff
@@ -1,10 +1,22 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 
-const routes: Routes = [];
+import { PageNotFoundComponent } from './shared/components/page-not-found/page-not-found.component';
+
+const routes: Routes = [
+  {
+    path: '',
+    loadChildren: () =>
+      import('./products/products.module').then((mod) => mod.ProductsModule),
```

Comment:
Please rename `ProductsModule` to something more general, like `shop/public/store/etc`. As we discussed on the lecture